contracts/OXO.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity ^0.4.15;
+
+contract OXO {
+ 
+    event BidEvent(address addr, uint ammount);
+    event MarkEvent(address addr, uint x, uint y);
+    event ErrorEvent(string message);
+    event WinnerEvent(uint8 winner);
+
+    address player1;
+    address player2;
+    uint player1Bid;
+    uint player2Bid;
+    uint8[3][] grid;
+    uint8 lastMarkByPlayer = 0;
+    uint lastMarkTimestamp = 0;
+    
+    function OXO() public {
+        grid = new uint8[3][](3);
```

Comment:
what is happening here?

Reply:
well, it's grid 3x3  :)   
(3) is redundant?

Reply:
Hm, I just thought it would be easier to understand if you write uint8[3][3], or it would have another meaning?

Reply:
if I do that remix complains: 
"browser/OXO.sol:20:20: TypeError: Length has to be placed in parentheses after the array type for new expression."
"browser/OXO.sol:20:16: TypeError: Type function (uint256) pure returns (uint8[3] memory[3] memory) is not implicitly convertible to expected type uint8[3] storage ref[] storage ref."